resources/lang/fr/wiki.php

```diff
@@ -31,5 +31,10 @@
             'link' => 'Afficher sur GitHub',
             'refresh' => 'Actualiser',
         ],
+
+        'outdated' => [
+            '_' => "Cette page est un ancienne traduction du contenu original. Veuillez vérifier la :default pour plus d'informations (and consider updating the translation if you are able to help out)!",
```

Comment:
(and consider updating the translation if you are able to help out)
tu as pas traduit cette partie